LINK/usr/bin/vmdb-up

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env ruby
+
+require "yaml"
+require "sd_notify"
+require "manageiq-password"
+
+$stdout.sync = true
+
+def wait_for_service(host, port)
+  loop do
+    `ncat #{host} #{port} </dev/null 2>/dev/null`
+    break if $?.success?
+
+    puts "#{host} #{port} - not accepting connections"
+    sleep(10)
+  end
+
+  puts "#{host} #{port} - accepting connections"
+end
+
+def wait_for_database(db_yaml, env = "production")
+  host, port = db_yaml[env].values_at("host", "port")
+  wait_for_service(host, port || 5432)
+end
+
+def wait_for_vmdb(db_yaml, env = "production")
+  host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
+  password = ManageIQ::Password.decrypt(password)
+
+  loop do
+    results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --command="SELECT FROM miq_regions LIMIT 1"`
+    break if $?.success? && results.include?("(1 row)")
+
+    puts "VMDB not ready yet"
```

Comment:
It's checking if it's properly seeded, right?  Maybe if this was the wording before, we could change this to reflect that the manageiq database isn't yet properly seeded.

Note, does this need to worry about  replication?  I don't know how you'd have remote regions in the global db without a local region but thought I'd ask since I had assumed that the original code used the local region scope.  Maybe I'm misremembering.

Reply:
I don't think it has to worry about replication, because we are only waiting for the database to be migrated and seeded. That will happen regardless of replication status.